_data/experiences/labor.yaml

```diff
@@ -0,0 +1,214 @@
+    title: Workers assemble Toyota Corollas at the New United Motor Manufacturing plant.
+    caption: 1999
+    date:
+    creator: David Bacon
+    collection: "Work & Social Justice: The David Bacon Photography Archive at Stanford"
+    note: As is often the case with labor photography, these images do not identify individual workers by name. However, they do provide a composite picture of the diversity of the NUMMI workforce and document the mix of manual and automated production at the plant.
+    link:
+    image: https://stacks.stanford.edu/image/iiif/sy022jy9750/autoworkers%20024/full/max/0/default.jpg
+    local_image: local-media/Content/Image-Derivatives/Slideshow-1-Screen-11-Labor/sy022jy9750.jpg
+  - key: workers-assemble-5
+    url: https://purl.stanford.edu/ks451yx7403
+    title: Workers assemble Toyota Corollas at the New United Motor Manufacturing plant.
+    caption:
+    date: 1999
+    creator: David Bacon
+    collection: "Work & Social Justice: The David Bacon Photography Archive at Stanford"
+    note: At its peak of productivity in 2006, workers at the NUMMI plant produced more than 400,000 vehicles.
+    link:
+    image: https://stacks.stanford.edu/image/iiif/ks451yx7403/autoworkers%20039/full/max/0/default.jpg
+    local_image: local-media/Content/Image-Derivatives/Slideshow-1-Screen-11-Labor/ks451yx7403.jpg
+  - key: workers-assemble-6
+    url: https://purl.stanford.edu/qz219sn3767
+    title: Workers assemble Toyota Corollas at the New United Motor Manufacturing plant.
+    caption:
+    date: 1999
+    creator: David Bacon
+    collection: "Work & Social Justice: The David Bacon Photography Archive at Stanford"
+    note: After GM withdrew from the partnership, Toyota continued to operate this facility, manufacturing Toyota Corollas there until the plant closed in 2009, bringing NUMMI’s history to an end.
+    link:
+    image: https://stacks.stanford.edu/image/iiif/qz219sn3767/autoworkers%20025/full/max/0/default.jpg
+    local_image: local-media/Content/Image-Derivatives/Slideshow-1-Screen-11-Labor/qz219sn3767.jpg
+  - key: workers-assemble-7
+    url: https://purl.stanford.edu/tq704xn6529
+    title: Workers assemble Toyota Corollas at the New United Motor Manufacturing plant.
+    caption:
+    date: 1999
+    creator: David Bacon
+    collection: "Work & Social Justice: The David Bacon Photography Archive at Stanford"
+    note: However, after a partnership was announced between Toyota and Tesla to develop electric vehicles, the plant was acquired by Tesla in 2010 and renamed the Tesla Fremont Factory.
+    link:
+    image: https://stacks.stanford.edu/image/iiif/tq704xn6529/autoworkers%20035/full/max/0/default.jpg
+    local_image: local-media/Content/Image-Derivatives/Slideshow-1-Screen-11-Labor/tq704xn6529.jpg
+  - key: farm-workers
+    url: https://purl.stanford.edu/gk561db9119
+    title: A crew of farm workers working behind a lettuce machine harvests head lettuce for D'Arrigo Brothers Produce
```

Comment:
these agricultural labor ones are kinda weird to me, because we have these long titles that duplicate the note or caption...i wonder if it would make more sense to have them all be titled "Agricultural Labor" and follow the same pattern as the manufacturing ones previously. these titles seem more like captions to me...i wonder if we should raise this at standup?

Reply:
@thatbudakguy I think you're 100% correct about the titles/captions; thanks for pointing it out. Feel free to bring up at standup if you want, though I'd suggest that this falls into the large bucket of "Text updates required by the curator" that we're going to schedule a dedicated meeting with Henry's team to discuss at some point, and knowing that, it might not be worth discussing at standup; in other words, we probably shouldn't resolve without Henry's okay, and knowing we'll have lots of related text things to discuss with him in the near future, I can just add this to the list and the Access team can just not worry about it for now.

Reply:
My preference would be (agreeing with @ggeisler) to merge this as is since it matches the spreadsheet and we can make changes as we get confirmation on what text updates are needed.

Reply:
thanks @ggeisler! glad to know there is a meeting planned.